website/siteConfig.js

```diff
@@ -25,17 +25,17 @@ const siteConfig = {
 
 	// (no longer needed, but keep just in case)
 	// UNCOMMENT FOR LOCAL DEVELOPMENT
-	//url: 'https://your-docusaurus-test-site.com', // Your website URL
-	//baseUrl: '/', // Base URL for your project
+	url: 'https://your-docusaurus-test-site.com', // Your website URL
+	baseUrl: '/', // Base URL for your project
 
 	// (no longer needed, but keep just in case)
 	// UNCOMMENT FOR PUBLISHING TO GH-PAGES
 	//url: 'https://1hive.github.io', // Your website URL
 	//baseUrl: '/website/', // Base URL for your project
 
 	// UNCOMMENT FOR PUBLISHING TO THE WEBSITE
-	url: 'http://1hive.org', // your website url
-	baseUrl: '/', // base url for your project
+	//url: 'http://1hive.org', // your website url
```

Comment:
I don't think this is necessary for deployments anymore?

Reply:
If it ain't broke, don't fix it lol